app/views/layouts/application.html.erb

```diff
@@ -11,7 +11,7 @@
 
   <body>
     <nav>
-      <%= link_to image_tag("H2L.png",width: 80, height: 60), { action: 'index', controller: 'welcome' } %>
+      <%= link_to image_tag("H2L.png",width: 80, height: 60), welcome_path %>
```

Comment:
Thanks for fixing this.
I wasn't sure how link_to and image_tag would work together, so I found this on the interwebs. Nice to know that's what it means.